### DISCUSSION TITLE
Question: Can I have two configurations stored in parallel?

### DISCUSSION THREAD

**Discussion by nefarius2001** [jamulussoftware/jamulus#998] [Technical Support & Help] [answered]
Hi,

I want to start 
- a first client with normal access to my USB-Audio-Interface
- a second client connected to a "virtual audio cable" (so i can play a virtual keyboard/metronome)

1) I know there is a --inifile-parameter, but have a hard time to find how to use it.
2) Can I give Profile-name & target audio-interface in a parameter/inifile for the "Jamulus"-call?

**Comment by chrisrimple** [comment, marked as answer]
I list the INI file locations for Windows and Mac in my [Remote Band Rehearsals](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/edit#heading=h.szc2tix6oj9l) doc.  On Windows, you can run multiple instances of Jamulus by changing the Shortcut parameters to use different INI files.  On Mac, you'll need to make a copy of the application, then run each copy from Terminal to specify the INI file that you want.

**Reply by nefarius2001** [reply]
thanks!
I'll give it a try soon.

Was there any chance to find this, starting on https://jamulus.io/ and clicking through liks there?

**Reply by pljones** [reply]
https://jamulus.io -> (right-hand) configuration -> https://jamulus.io/wiki/Command-Line-Options -> 
| -i | --inifile | Set location of initialization file (overrides default)  |

But I guess it would help to have something in "Tips and Tricks" explaining that everything that gets set up on https://jamulus.io/wiki/Software-Manual and https://jamulus.io/wiki/Server-Win-Mac are saved in an inifile and that `--inifile` allows you to run with different set ups.

Now, whether you can have two processes accessing the same device at the same time depends on that device, of course.  On Windows, some hardware ASIO drivers don't let you whilst some do; most virtual ASIO drivers do - that being part of the point.  It's not an issue on Linux (or other OSes using it) - JACK happily handles multiple clients.  I don't know MacOS, though.
